executor/select_into.go

```diff
@@ -193,7 +193,15 @@ func (s *SelectIntoExec) dumpToOutfile() error {
 			case mysql.TypeJSON:
 				s.fieldBuf = append(s.fieldBuf, row.GetJSON(j).String()...)
 			}
-			s.lineBuf = append(s.lineBuf, s.escapeField(s.fieldBuf)...)
+
+			switch col.GetType().Tp { // only string types need to be escaped
+			case mysql.TypeString, mysql.TypeVarString, mysql.TypeVarchar,
```

Comment:
Can we check whether col.GetType.EvalType is ETString or ETJSON here?

Reply:
Good suggestion.
It is updated.